CI: TestNodeChurnXFRMLeaksEncryptedOverlay `Should NOT be empty, but was []`

## CI failure

```
FAIL  package: github.com/cilium/cilium/pkg/datapath/linux
--- FAIL: TestNodeChurnXFRMLeaksEncryptedOverlay (2.06s)

time="2025-02-19T16:45:26.312182659Z" level=debug msg="Registered BPF map" path=/sys/fs/bpf/tc/globals/test_cilium_tunnel_map subsys=bpf
time="2025-02-19T16:45:27.328763333Z" level=debug msg="Updating tunnel map entry" endpoint=3.3.3.3 key=0 prefix=4.4.4.0 subsys=map-tunnel
time="2025-02-19T16:45:28.329030346Z" level=debug msg="Silently deleting tunnel map entry" prefix="2001:aaaa::" subsys=map-tunnel
    node_linux_test.go:881: 
        	Error Trace:	/host/pkg/datapath/linux/node_linux_test.go:881
        	            				/host/pkg/datapath/linux/node_linux_test.go:818
        	Error:      	Should NOT be empty, but was []
        	Test:       	TestNodeChurnXFRMLeaksEncryptedOverlay
```

https://github.com/cilium/cilium/actions/runs/13417441188/job/37481832373#step:17:17517

unfortunately no sysdumps:
> No matching cilium-sysdumps-* artifacts found.

--- a/pkg/datapath/linux/node_linux_test.go
+++ b/pkg/datapath/linux/node_linux_test.go
@@ -1,3 +1,5 @@
+//go:build unparallel
+
 // SPDX-License-Identifier: Apache-2.0
 // Copyright Authors of Cilium
 

